### PR TITLE
2021-11 update api versions and ingress

### PR DESCRIPTION
**Pull request by Hisdad**
Works cleanly. the ingress was previously broken entirely

### CODE REVIEW

File: charts/maildev/templates/ingress.yaml [hunk 1]
```diff
@@ -11,26 +11,30 @@ metadata:
     {{- toYaml . | nindent 4 }}
   {{- end }}
 spec:
+  ingressClassName: {{ .Values.ingress.spec.ingressClassName | quote}}
```
Comment (audig) [inline charts/maildev/templates/ingress.yaml:14]: Can set this block conditionnal : if variables is not empty in values.yaml

File: charts/maildev/values.yaml [hunk 2]
```diff
@@ -53,11 +54,15 @@ cron:
 
 ingress:
   enabled: false
+  spec:
+  # ingressClassName is standard for 1.19+
+    ingressClassName: nginx
```
Comment (audig) [inline charts/maildev/values.yaml:59]: Can you set this values to empty by default ?

**Review by audig — CHANGES_REQUESTED** [review]

File: charts/maildev/templates/ingress.yaml (hunk 1 above)
Comment (Hisdad) [reply charts/maildev/templates/ingress.yaml:15]: Done

File: charts/maildev/values.yaml (hunk 2 above)
Comment (Hisdad) [reply charts/maildev/values.yaml]: And also done, hmmm how do I progress this..

**Review by audig — APPROVED** [review]

**Pull request merged**